scripts/network_bandwidth.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+INTERVAL="1"  # update interval in seconds
+
+network_name=$(tmux show-option -gqv "@dracula-network-bandwidth")
+
+download_bytes() {
+  case $(uname -s) in
+    Linux)
+      cat "/sys/class/net/$network_name/statistics/rx_bytes"
+      ;;
+    Darwin)
+      netstat -I "$network_name" -b | tail -n 1 | awk '{print $7}'
+      ;;
+    CYGWIN*|MINGW32*|MSYS*|MINGW*)
+      # TODO - windows compatibility
+      ;;
+  esac
+}
+
+upload_bytes() {
+  case $(uname -s) in
+    Linux)
+      cat "/sys/class/net/$network_name/statistics/rx_bytes"
```

Comment:
This should be `tx_bytes`

Reply:
Whats the difference between the two?

Reply:
rx is for received bytes, tx is for transmitted

Reply:
I see, makes sense.